### ISSUE TITLE
Don't run on cypress open

### ISSUE THREAD

**Issue by fr0** [Bkucera/cypress-plugin-retries#51]
I think this plugin should only take effect on `cypress run`, not on `cypress open`.
When using `cypress open`, having the plugin makes it harder to debug problems because it starts repeating the test after an error and scrolls past the error.

**Comment by Bkucera** [comment]
@fr0 agreed. I'll turn that off by default and add a configuration variable to turn it back on

**Comment by dialex** [comment]
Any update on this? I just finished installing and configuring it and the tests simply don't run on `cypress open`. If I do `cypress run` then tests run and I can see retries as expected. There are no errors on console or browser.

**Comment by Bkucera** [comment]
@dialex if you remove the retries plugin does cypress work as normal?

**Comment by dialex** [comment]
Oh no, you're right, even without this plugin the tests don't run on `cypress open` 😵😱 

> Offtopic: I'm using Cypress 3.7.0 and I did not change anything about it. But Chrome recently updated to v80. Could it be a conflict between versions? Because the tests do run if I choose Electron v73.

**Comment by Bkucera** [comment]
@dialex I believe its a problem with Chrome 80.
